lib/takeaway.rb

```diff
@@ -0,0 +1,43 @@
+require_relative 'menu.rb'
+require_relative 'order.rb'
+require_relative 'twilio.rb'
+
+class Takeaway
+
+attr_accessor :final_order
+
+  def initialize(menu=Menu.new)
+    @menu = menu
+    @order = Order.new
+    @message = Message.new
+  end
+
+  def show_menu
+    @menu.menu_list
+  end
+
+  def order(dish, number=1)
+    fail "Item not on the menu: please choose again" unless @menu.menu_list.include? dish
+    @order.take_order(dish, number, @menu)
+  end
+
+  def basket
+    @order.basket
+  end
+
+  def complete_order(price)
+    fail 'Incorrect amount' if is_payment_correct?(price) != true
+    send_text("Your order totalling $#{@order.total} has been registered and will be delivered before 18:52")
+    @order.current_order.clear
+  end
+
+  private
+
+  def send_text(message)
+    @message.send_message(message)
+  end
+
+  def is_payment_correct?(price)
```

Comment:
Style/PredicateName: Rename `is_payment_correct?` to `payment_correct?`. (https://github.com/bbatsov/ruby-style-guide#bool-methods-qmark)